Tools/Pipeline/Windows/MainView.cs

```diff
@@ -409,7 +413,8 @@ public void AddTreeItem(IProjectItem item)
             node.ImageIndex = iconIdx;
             node.SelectedImageIndex = iconIdx;
 
-            _treeView.SelectedNode = node;
+            if (!_projectLoading)
+                _treeView.SelectedNode = node;
```

Comment:
Correct. When the tree root is set (the project) it is selected. Hence it is already in the state we want, we just have to disable the normal behavior of selecting subsequent items that are added during load (after load, you DO want new items added to be selected).